source/usage-examples/count.txt

```diff
@@ -9,14 +9,17 @@ collection by using the ``EstimatedDocumentCount()`` method and an exact
 number of documents in a collection by using the ``CountDocuments()``
 method.
 
-The following example uses the ``EstimatedDocumentCount()`` method to
-count the number of documents in the ``movies`` collection. Then, the
-example uses the ``CountDocuments()`` method to count the number of
-documents in the ``movies`` collection where the ``countries`` field
-contains "China".
+Example
+-------
 
 .. include:: /includes/usage-examples/run-example-tip.rst
 
+The following example uses the ``EstimatedDocumentCount()`` method to
+count the number of documents in the ``movies`` collection. Then, the
```

Comment:
Personally, I'm not sure the user or writer gets any value from standardizing the structure of this sentence for all usage examples, but I think this example and a couple others do not match the one in your PR description:

> Updated all example sections to the following:
> The example statement with the format: “The following example passes .... to the ___ method, which ...”

Applies to some of the updated instances.